src/components/DropdownItem/index.tsx

```diff
@@ -8,11 +10,22 @@ export interface IDropdownItemProps {
 }
 
 const DropdownItem: FC<IDropdownItemProps> = ({ propertyKey, propertyValue }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  const onItemClick = useCallback(() => {
+    navigator.clipboard.writeText(`${propertyKey}: ${parametersToString(propertyValue)}`);
+    setIsCopied(true);
+
+    setTimeout(() => {
+      setIsCopied(false);
+    }, COPY_ANIMATION_DURATION);
+  }, [propertyKey, propertyValue]);
+
   return (
-    <li className="CardPanel">
-      <p>
+    <li className="CardPanel" onClick={onItemClick}>
+      <p className={classNames({ 'CardPanel_Copied': isCopied })}>
         {`${propertyKey}: `}
-        <span>{parametersToString(propertyValue)}</span>
+        <span className={classNames({ 'CardPanel_Copied': isCopied })}>{parametersToString(propertyValue)}</span>
```

Comment:
Can you make that changing color after a click will be only to value not to key?